extension/app/ts/utils/ethereum.ts

```diff
@@ -50,15 +50,25 @@ export interface IUnsignedTransaction1559 {
 	}[]
 }
 
-export interface ITransactionSignature {
+export interface ITransactionSignatureLegacy {
+	readonly r: bigint
+	readonly s: bigint
+	readonly v: bigint
+	readonly hash: bigint
+}
```

Comment:
Internally, I personally prefer storing all transaction signatures as `r,s,yParity` and only converting to `v` at (de)serialization time.  This helps keep things consistent even though the wire protocol is insane.